core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -880,20 +846,6 @@
           + " general.vfs.context.classpath.<name>.delegation=post, where <name> is"
           + " your context name. If delegation is not specified, it defaults to loading"
           + " from parent classloader first."),
-  @Interpolated
-  VFS_CLASSLOADER_CACHE_DIR(AccumuloVFSClassLoader.VFS_CACHE_DIR,
```

Comment:
Like the other VFS property, I think it's still in use via `AccumuloVFSClassLoader.VFS_CACHE_DIR`, but exists here for documentation.

Reply:
Ah I see how I missed those usages. Fixing in new commit